components/table/demo/table-test.js

```diff
@@ -0,0 +1,61 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+import { RtlMixin } from '../../../mixins/rtl-mixin.js';
+import { tableStyles } from '../table-wrapper.js';
+
+class TestTable extends RtlMixin(LitElement) {
+
+	static get properties() {
+		return {
+			type: { type: String },
+			stickyHeaders: { attribute: 'sticky-headers', type: Boolean }
+		};
+	}
+
+	static get styles() {
+		return tableStyles;
```

Comment:
By the way, I did confirm that the imported styles end up in their own constructed/adopted stylesheet that is shared across usages.  And FWIW, it is also possible to provide a constructed stylesheet here as well - I _think_ it was Lit-element version 2.4 that added that capability.